caikit_tgis_backend/tgis_connection.py

```diff
@@ -94,12 +96,22 @@ def from_config(cls, model_id: str, config: dict) -> Optional["TGISConnection"]:
         """Create an instance from a connection template and a model_id"""
         hostname = config.get(cls.HOSTNAME_KEY)
         if hostname:
-            hostname = hostname.format(
-                **{
+            assert isinstance(hostname, str)
+            # TODO: Namespace needs to be populated here as well into the hostname template
+            namespace = config.get(cls.NAMESPACE_KEY)
+
+            hostname = hostname.format_map(
+                {
                     cls.HOSTNAME_TEMPLATE_MODEL_ID: model_id,
+                    cls.HOSTNAME_TEMPLATE_NAMESPACE: namespace,
```

Comment:
My concern here is with existing models (eg. PT models) who's config.yaml don't already contain the namespace info.

If the provided `hostname` template string includes a `{namespace}` param to be replaced, but the config doesn't contain the namespace info, then we break backward compatibility.

Reply:
@gabe-l-hart